<launch>
  <arg name="gt_file" default = "$(find object_segmentation)/config/gt_obj.txt" />
  <arg name="save_obj_file" default = "$(find object_segmentation)/config/result_obj.txt" />
  <node name="show_hypermap_node" pkg="object_segmentation" type="show_hypermap_node" output="screen">
    <param name="save_obj_file"                  value="$(arg save_obj_file)" />
    <param name="gt_file"                  value="$(arg gt_file)" />
  </node>

  <!-- rviz -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find object_segmentation)/rviz/hypermap.rviz" />

  <!-- Map server -->
  <arg name="map_file" default="$(find object_segmentation)/maps/map.yaml"/>
  <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)"/>




</launch>
